addon/components/rdfa/editor-toolbar.hbs

```diff
@@ -46,15 +46,15 @@
       </button>
       {{/if}}
       {{#if @showListButtons}}
-      <button type="button" class="say-toolbar__button" {{on "click" this.insertUL}} data-test-button-id="unordered-list">
+      <button type="button" class={{concat "say-toolbar__button" (if this.isInList "active")}} {{on "click" this.toggleUnorderedList}} data-test-button-id="unordered-list">
```

Comment:
it's probably best to not use concat here. there proper way of doing this is:
`<button class="say-toolbar__buton {{if this.isInList "active"}}" ...>`

Reply:
ah ok ill change it, I actually thought concat was better